### DISCUSSION TITLE
My application was rejected for location error

### DISCUSSION THREAD

**Discussion by Zhong-Mingyu** [community/community#53001] [GitHub Education]
Hello there!
I have applied for education pack for several times, but it is still not verified. I have submitted my school ID and the online vertification report of academic registration from the Ministry of Education of my country.
Firstly, according to email sent to me, I have to complete my Github user profile and plus a short bio, I have done that following the tutorial sent to me, but it is not solved. I wrote my school in my public profile, my real name showed on document submitted, and a short bio showing that I'm studying in my school.
Secondly, email tells me that I'm not near any campus location for the school I have selected, but I am still living in my school for now. **Very strange!!**
Internet connection in my school sometimes doesn't have access to GitHub, so I have to use SSH to access GitHub, but it won't influence my GPS location. At the same time, I have applied once without using SSH, email still shows that I'm not near any campus location for the school I have selected, it's very weird.
Therefore, I try to ask for help in here. I found that there are some friends who met similar problems to me, but there is a difference that the email sent to me didn't show that GitHub doesn't have the location of my school, so why it said that I'm not near my school??

**Thanks for anyone's kind and help!!**

**Comment by jayden-n** [comment]
Hello,

It seems you're facing challenges with verifying your GitHub Education Pack application. While I can't access your account, I can offer some general advice:

1. Contact Support: Reach out to GitHub Support for personalized assistance.

2. Profile Accuracy: Ensure your public GitHub profile matches submitted documents.

3. IP and GPS: GitHub might use IP addresses for location; explain any discrepancies.

4. Provide Details: When contacting support, share all relevant information.

5. Be Patient: Verification can take time; work with GitHub Support for resolution.

I hope this helps!
